flake.lock

```diff
@@ -33,18 +39,35 @@
     },
     "root": {
       "inputs": {
-        "import-cargo": "import-cargo",
+        "fenix": "fenix",
         "nixpkgs": "nixpkgs",
         "utils": "utils"
       }
     },
+    "rust-analyzer-src": {
+      "flake": false,
+      "locked": {
+        "lastModified": 1636846936,
+        "narHash": "sha256-3xJCEfFdYkdxrQ3MP20ZbcjOBPxuG8m89WxpC4WkeW0=",
+        "owner": "rust-analyzer",
+        "repo": "rust-analyzer",
+        "rev": "766b52b598ad1639e5b815b87dbd3fa3be155219",
+        "type": "github"
```

Comment:
This seems like quite a big dependency - even though it's only for dev purposes (yes, I live in Germany, we don't have fast Internet here %)). Is there any chance to strip that down a bit?

To describe my issue: I'm using direnv and I have distributed building enabled by default (since it mostly saves time). And after ~25mins I decided to stop waiting for my shell to be ready.

Also regarding what needs to be built: is there any way to cache this?

Reply:
IMHO `fenix` should just be caching this to the nix-community cachix instance. I'll ask around how to make that happen.

Reply:
There are a few options, one being adding adding the nix-community substituter to the system config, other adding the nix-community url to nixConfig.extra-substituters in the flake.nix, and the last making a `nix --extra-substituters url print-dev-env` in direnv